Formula/glade.rb

```diff
@@ -1,42 +1,38 @@
 class Glade < Formula
   desc "RAD tool for the GTK+ and GNOME environment"
   homepage "https://glade.gnome.org/"
-  url "https://download.gnome.org/sources/glade3/3.8/glade3-3.8.5.tar.xz"
-  sha256 "58a5f6e4df4028230ddecc74c564808b7ec4471b1925058e29304f778b6b2735"
-  revision 1
-
-  bottle do
-    sha256 "5c910f1666f6e261369aa166bea2e48e5b03856928900448c2c577ec4f04eca2" => :el_capitan
-    sha256 "8f0c1d019b71e76381c0f4437395f1e1eb3da24e8ef1b16051535209aeeaf5d7" => :yosemite
-    sha256 "f4eae837311b7aadfff8a27720ce2bbc71871e9b095d31e90d4b833ec725205f" => :mavericks
-  end
+  url "https://download.gnome.org/sources/glade/3.20/glade-3.20.0.tar.xz"
+  sha256 "82d96dca5dec40ee34e2f41d49c13b4ea50da8f32a3a49ca2da802ff14dc18fe"
 
   depends_on "pkg-config" => :build
   depends_on "intltool" => :build
+  depends_on "itstool" => :build
+  depends_on "docbook-xsl" => :build
   depends_on "gettext"
   depends_on "libxml2"
+  depends_on "gnome-icon-theme"
   depends_on "hicolor-icon-theme"
-  depends_on "gtk+"
+  depends_on "gtk+3"
   depends_on "gtk-mac-integration"
 
-  # patch restores compatibility with the latest gtk-mac-integration version
-  # this bug has been filed: https://bugzilla.gnome.org/show_bug.cgi?id=730778
-  patch :DATA
-
   def install
     # Find our docbook catalog
     ENV["XML_CATALOG_FILES"] = "#{etc}/xml/catalog"
 
     system "./configure", "--disable-debug",
                           "--disable-dependency-tracking",
-                          "--prefix=#{prefix}"
+                          "--prefix=#{prefix}",
+                          "--enable-gladeui"
+    # objective-c is needed for glade-registration.c. unfortunately build fails if -x objective-c is added to global CFLAGS.
+    inreplace "src/Makefile", "-c -o glade-glade-registration.o", "-x objective-c -c -o glade-glade-registration.o"
```

Comment:
If this patch was submitted upstream to their bugzilla could you make that an inline comment pointing to it in the Ruby file too? Thanks.